ydb/docs/en/core/concepts/datamodel/view.md

```diff
@@ -0,0 +1,76 @@
+# View
+
+A view is a query which is treated as if it was a table storing the results of a given query. The view itself contains no data. The content of a view is generated every time you `SELECT` from it. Thus, any changes in the underlying tables are reflected immediately in the view.
```

Comment:
```suggestion
A view is a query that is treated as if it were a table storing the results of a given query. The view itself contains no data. The content of a view is generated every time you `SELECT` from it. Thus, any changes in the underlying tables are reflected immediately in the view.
```

Reply:
I didn't know about this "were" usage case. Thanks!